generate_utils.py

```diff
@@ -122,7 +125,33 @@ def generate_primary_key_data(column, table, seed):
     return
 
 def generate_column_data(column, table, seed):
-    return
+    columnType = column["type"]
+    
+    # isUnique = column.get("isUnique", False) This is difficult, what if the user wants uniform distribution min max with unique values
+    isNullable = column.get("isNullable", False)
+    percentageNull = column.get("percentageNull", 0)
+    if not isNullable and percentageNull > 0:
+        raise ValueError("Column is not nullable but percentageNull > 0")
+    rows = table["numRows"]
+    numRowsToSample = math.floor(rows * (1 - percentageNull))
+    if is_number_type(columnType):
+        sampled_answer_row = distribution.generate_integer_distribution(column, numRowsToSample)
+    elif columnType == "text":
+        args = {}
+        if "constraints" in column:
+            args = column["constraints"]
+        sampled_answer_row = text_generation.generate_text_column(numRowsToSample, **args)
+    
+    if isNullable:
+        null_array = ["null" for i in range(rows - numRowsToSample)]
+        string_list = [str(item) for item in sampled_answer_row]
+        final_result_array = null_array + string_list
+        np.random.shuffle(final_result_array)
+        return final_result_array
+    return sampled_answer_row
```

Comment:
I think you forgot to use the seed to generate data

Reply:
Yup, added the seed